### ISSUE TITLE
App not installing after install this package

### ISSUE THREAD

**Issue by viveKing21** [ammarahm-ed/react-native-admob-native-ads#193]
```
> Task :app:processDebugManifest FAILED

See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.


Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.2/userguide/command_line_interface.html#sec:command_line_warnings
233 actionable tasks: 3 executed, 230 up-to-date
[com.facebook.android:audience-network-sdk:6.5.1] C:\Users\sharm\.gradle\caches\transforms-2\files-2.1\3224c4623f334ce6aa4180b86f5aac2e\audience-network-sdk-6.5.1\AndroidManifest.xml:12:9-55 Error:
        Missing 'package' key attribute on element package at [com.facebook.android:audience-network-sdk:6.5.1] AndroidManifest.xml:12:9-55
[com.facebook.android:audience-network-sdk:6.5.1] C:\Users\sharm\.gradle\caches\transforms-2\files-2.1\3224c4623f334ce6aa4180b86f5aac2e\audience-network-sdk-6.5.1\AndroidManifest.xml Error:
        Validation failed, exiting

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed with multiple errors, see logs

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup. Run CLI with --verbose flag for more details.
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081
[com.facebook.android:audience-network-sdk:6.5.1] C:\Users\sharm\.gradle\caches\transforms-2\files-2.1\3224c4623f334ce6aa4180b86f5aac2e\audience-network-sdk-6.5.1\AndroidManifest.xml:12:9-55 Error:
```

i got this error after adding this package on my app

I installed 0.3.6 version and its working, can you please fix this on latest version too

**Comment by wjaykim** [comment]
What react native version and your android project's gradle version are you using?

**Comment by viveKing21** [comment]
> What react native version and your android project's gradle version are you using?

React native  0.63.4, React 16.13.1, Gradle 6.2

**Comment by wjaykim** [comment]
> > What react native version and your android project's gradle version are you using?
> 
> React native 0.63.4, React 16.13.1, Gradle 6.2

What's your gradle version, not your gradle wrapper version?

**Comment by ammarahm-ed** [comment]
Fixed v0.5.0
